includes/class-affiliatewp-leaderboard.php

```diff
@@ -0,0 +1,366 @@
+<?php
+/**
+ * Core: Plugin Bootstrap
+ *
+ * @package     AffiliateWP Leaderboard
+ * @subpackage  Core
+ * @copyright   Copyright (c) 2021, Sandhills Development, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       1.0.0
```

Comment:
The file is new, the code is not.
```suggestion
 * @since       1.1
```